es-design-system/pages/examples/form-validation.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+    <b-container>
+        <b-row>
+            <b-col>
+                <h1>
+                    Form with (form level) Validation
+                </h1>
+                <p>
+                    See
+                    <b-link
+                        :to="{ name: 'examples-form-field-validation___en' }">
+                        Form Field Validation
+                    </b-link> for field level validation example
+                </p>
+                <h2>
+                    UX Guidelines
+                </h2>
+                <p>
+                    <ul>
+                        <li>
+                            Show loading spinner on button during server request
+                        </li>
+                        <li>
+                            Use <b-link :to="{ name: 'molecules-es-form-msg___en' }">
+                                EsFormMsg
+                            </b-link>
+                            for displaying of server-side form-level errors
+                        </li>
+                    </ul>
+                </p>
+            </b-col>
+        </b-row>
+        <b-row class="border-top pt-4 my-2">
+            <b-col
+                cols="12"
+                lg="8">
+                <b-form
+                    @submit.stop.prevent="onSubmit">
+                    <es-form-msg
+                        class="mt-3"
+                        :variant="formMsgVariant"
+                        :message="formMsg"
+                        @hidden="formMsg = ''" />
+                    <es-form-input
+                        id="form-input-name"
+                        v-model="form.name"
+                        :disabled="isSubmitInProgress"
+                        required>
+                        <template #label>
+                            Name
+                        </template>
+                    </es-form-input>
+                    <div class="d-flex flex-grow-1 justify-content-end mt-3">
+                        <es-button
+                            type="submit"
+                            class="w-100 w-lg-auto"
+                            :loading="isSubmitInProgress"
+                            :disabled="isSubmitInProgress">
+                            <span class="w-100 min-width-6">
+                                <span
+                                    v-if="isSubmitInProgress">
+                                    <b-spinner
+                                        role="status"
+                                        aria-hidden="true"
```

Comment:
Do we need this `aria-hidden`? If we're going to the trouble of including a `label`, we should probably let the browser announce the label to users via the `role="status"` and the `label` rather than hiding it.

Reply:
Yup, good catch, fixed via 616bb970e812d7ae4364c2aa25fa970a50b1cc65